kweeuhree/react-website
# Offload Video Hosting to Cloudinary

This PR addresses the issue of offloading video hosting to Cloudinary to improve performance and user experience. By storing and serving videos through Cloudinary, we ensure faster media delivery and reduce server load.

### Changes

- Integrated Cloudinary for video hosting in the application.
- Updated logic for video upload and retrieval to work with Cloudinary.
- Replaced local video URLs with Cloudinary URLs.
- Added video optimization using Cloudinary's transformation features.
- Updated `.env` file to include necessary Cloudinary environment variables.

### Closes

Closes #20 

### Acceptance Criteria

- [x] Videos are uploaded and stored in Cloudinary.
- [x] Video playback uses Cloudinary URLs.
- [x] Video quality adjustments are handled via Cloudinary transformations.
- [x] Environment variables for Cloudinary are properly set up.

### Testing Instructions

1. Upload a video through the application.
2. Verify the video is uploaded to Cloudinary and can be played using the Cloudinary URL.
3. Test video performance across different devices and resolutions.
4. Ensure video transformations like resolution changes are working as expected.
